utils/publicSettings/publicSettings.go

```diff
@@ -237,6 +239,18 @@ func GetMascotURLFromRequest(c *gin.Context) string {
 	return ""
 }
 
+func GetEUCookieFromRequest(c *gin.Context) bool {
+	cookie, err := c.Cookie("EU_Cookie")
```

Comment:
Since you don't use cookie, you just have to do:
_, err := c.Cookie("EU_Cookie")